tests/mock/performance_mocks.py

```diff
@@ -90,6 +89,10 @@ def __eq__(self, other):
 
 class NotAPrivateKey:
 
+    fake_signature = Signature.from_bytes(
```

Comment:
Is it a rebase thing, or it's actually used somewhere?

Reply:
✔️